design-documents/graph-ql/coverage/reward-points.graphqls

```diff
@@ -0,0 +1,70 @@
+type Customer {
+    reward_points: RewardPoints @doc(description: "Customer reward points details")
+}
+
+type Cart {
+    applied_reward_points: RewardPointsAmount @doc(description: "Reward points ammount applied to the cart")
+}
+
+type Mutation {
+    applyRewardPointsToCart(cartId: ID!): ApplyRewardPointsToCartOutput @doc(description: "Apply all available points up to the cart total. Partial redemption is not available")
+    removeRewardPointsFromCart(cartId: ID!): RemoveRewardPointsFromCartOutput @doc(description: "Cancel usage of the previously applied reward points to cart")
```

Comment:
Probably not the end of the world, but the symmetry here seems off with the naming

- `apply`
- `remove`

I think `apply` makes sense, since you're not conceptually _adding_ to the cart.

But if you're not `adding` something, you wouldn't expect to be able to `remove` that thing.

tl;dr: I think `apply` and `remove` don't work well together, but I'm not sure of a better alternative

Reply:
This the the naming we currently have for coupons and gift cards.. not sure whether there is a good alternative. Also seems better to keep similar naming for consistency.

Reply:
> Also seems better to keep similar naming for consistency.

I think this could be argued in both directions - depends on whether you're optimizing for folks new to the API, or folks familiar with the existing API surface.

But I also can't think of better wording so it's a moot point 😂. Makes sense to me to just move forward by matching what we have 👍